client/src/pages/admin/MergePeople.js

```diff
@@ -170,19 +171,25 @@ class MergePeople extends Page {
                   </Row>
                   <Row>
                     <Col md={12}>
-                      {loser && loser.position && winner && !winner.position && (
+                      {loser &&
+                        !_isEmpty(loser.position) &&
+                        winner &&
+                        _isEmpty(winner.position) && (
                         <Field
                           name="copyPosition"
                           component={FieldHelper.renderSpecialField}
                           label={null}
                           widget={
                             <Checkbox inline checked={values.copyPosition}>
-                              Set position on winner to {loser.position.name}
+                                Set position on winner to {loser.position.name}
```

Comment:
Too much leading space?

Reply:
Unfortunate side-effect of the Javascript formatter I'm afraid. Nothing can be done…

Reply:
Alrigth, I though it was weird the linter did not catch it.